en/mkdocs.yml

```diff
@@ -801,7 +801,6 @@ nav:
                 - Developer Portal API v2: reference/product-apis/devportal-apis/devportal-v2/devportal-v2.md
             - Admin APIs:
                 - Admin APIs v2: reference/product-apis/admin-apis/admin-v2/admin-v2.md
-                - Admin APIs v0.17: reference/product-apis/admin-apis/admin-v0.17/admin-v0.17.md
```

Comment:
If this is being removed, you need to update page [1] as well.

[1] https://apim.docs.wso2.com/en/latest/reference/product-apis/overview/

Reply:
Already updated https://github.com/wso2/docs-apim/pull/3970/files#diff-9772aa32fad308a59ad99ee3ca114c0618154183887e27d0b287985ecf02c787L25-L28

Reply:
Sorry, my bad!